### PR TITLE
[GNA] Fix concat scale factors calculation

### PR DESCRIPTION
**Pull request by kbruniec**
### Details:
 - One liner to fix cnn2d hanging
 - Add test

### Tickets:
 - 53519

### CODE REVIEW

**Comment by kbruniec** [comment]
@elilobanova The test reproducing the hanging was added. It would be great if you could review. Thanks.

File: inference-engine/tests/functional/plugin/shared/include/subgraph_tests/const_strided_slice_concat.hpp [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+//
+
+#pragma once
+
+#include "shared_test_classes/subgraph/const_strided_slice_concat.hpp"
+
+namespace SubgraphTestsDefinitions {
+
+TEST_P(ConstStridedSliceConcatTest, CompareWithRefImpl) {
+    LoadNetwork();
```
Comment (elilobanova) [inline inference-engine/tests/functional/plugin/shared/include/subgraph_tests/const_strided_slice_concat.hpp:12]: Why don't you just call Run()?

Comment (kbruniec) [reply inference-engine/tests/functional/plugin/shared/include/subgraph_tests/const_strided_slice_concat.hpp:12]: Good point. I will simplify with Run(). Thanks

**Comment by kbruniec** [comment]
> 
> 
> @elilobanova The test reproducing the hanging was added. It would be great if you could review. Thanks.

@elilobanova Gentle ping for approval/comments, as your previous review was applied and CI is passing :).

**Review by elilobanova — APPROVED** [review]

**Pull request merged**